### PR TITLE
Building user profile page

### PR DESCRIPTION
**Pull request by kcalva**

### CODE REVIEW

**Review by CodeX-y — APPROVED** [review]
Looks good. Your users show page looks right and you changed your comments controller appropriately.

**Review by ghost — APPROVED** [review]
It's lookin good to me!

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -11,22 +11,12 @@
...
-
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
-
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:14]: Nice use of hidden input!

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <%photo.comments.each do |comment|%>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:20]: This could be removed if its not being used or you could add default values :)

File: config/routes.rb [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+  # Defines the root path route ("/")
+  # root "articles#index"
   root "photos#index"
   devise_for :users
 
   resources :likes
   resources :follow_requests
   resources :comments
   resources :photos
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:11]: Nice!

**Review by JoseMDO — APPROVED** [review]
Looks good Kevin!

File: app/views/users/show.html.erb
Comment (rayguna) [inline app/views/users/show.html.erb:1]: A neat use of refactoring with views/_form.html.erb.

**Pull request merged**